spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -90,7 +90,9 @@
         "spark",
         SparkInterpreter.class.getName(),
         new InterpreterPropertyBuilder()
-            .add("spark.app.name", "Zeppelin", "The name of spark application.")
+            .add("spark.app.name",
```

Comment:
Sure. If you are to set SPARK_APP_NAME in zeppelin-env.sh, you could also set spark.app.name in interpreter.json also in /conf?